presto-main/src/main/java/com/facebook/presto/security/FileBasedSystemAccessControl.java

```diff
@@ -87,17 +86,22 @@ public String getName()
         @Override
         public SystemAccessControl create(Map<String, String> config)
         {
+            final String CONFIG_FILE_NAME = "security.config-file";
             requireNonNull(config, "config is null");
-            Path path = Paths.get(config.get("security.config-file"));
-            if (!path.isAbsolute()) {
-                path = path.toAbsolutePath();
-            }
+
             try {
+                Path path = Paths.get(config.get(CONFIG_FILE_NAME));
```

Comment:
The whole idea of ConfigurationNotFoundException class was for handling that. It would throw an exception if that entry isn't found. Anyway, I am changing this now to add a `checkState` instead.